test/unit/proc_macro/proc_macro_2015.rs

```diff
@@ -0,0 +1,10 @@
+extern crate proc_macro;
+
+use proc_macro::TokenStream;
+
+/// This macro is a no-op; it is exceedingly simple as a result
+/// of avoiding dependencies on both the syn and quote crates.
+#[proc_macro_derive(HelloWorld)]
+pub fn hello_world(_input: TokenStream) -> TokenStream {
+    TokenStream::new()
```

Comment:
Suggestion: you can `return input;` so the parameter is used. It would then also fit the Noop name better. Up to you.

Reply:
[same action](https://github.com/bazelbuild/rules_rust/pull/807#discussion_r662070938)